contentcuration/contentcuration/frontend/channelEdit/vuex/contentNode/actions.js

```diff
@@ -425,11 +432,15 @@ export function updateContentNode(context, { id, mergeMapFields, ...payload } =
     ...node,
     ...contentNodeData,
   };
-  const complete = isNodeComplete({
-    nodeDetails: newNode,
-    assessmentItems: context.rootGetters['assessmentItem/getAssessmentItems'](id),
-    files: context.rootGetters['file/getContentNodeFiles'](id),
-  });
+  let complete = true;
```

Comment:
If we're not checking complete, we should just not set this attribute at all, either true or false.